nodes/src/nodes/audio_tts/IGlobal.py

```
@@ -0,0 +1,273 @@
+
+        from depends import depends  # type: ignore
+
+        requirements = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'requirements.txt')
+        depends(requirements)
+
+        from .tts_engine import TTSEngine
+
+        self._config = self._build_tts_config_dict()
+        self._engine = TTSEngine(self._config)
+
+    def _cleanup_stale_outputs(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_cleanup_stale_outputs`

Add return type annotation: `None`

(ANN202)